native/src/routes/SearchModal.tsx

```diff
@@ -22,6 +22,12 @@ const Wrapper = styled.View`
   background-color: ${props => props.theme.colors.backgroundColor};
 `
 
+const SearchCounter = styled.Text`
+  margin: 10px 20px;
+  fontsize: 'medium';
```

Comment:
Same as below:
1) should be "font-size" 
2) please use the value from the theme

Not quite sure, but I think if you just want to use the "normal" font size, you do not have to specify it.